scripts/webpack/prod.servers.config.js

```diff
@@ -88,9 +87,22 @@ module.exports = ({isDeploy}) => ({
         test: /\.(png|jpg|jpeg|gif|svg)$/,
         use: [
           {
-            loader: 'file-loader'
+            loader: 'file-loader',
+            options: {
+              publicPath
+            }
           }
         ]
+      },
+      {
+        include: [/node_modules/],
+        test: /\.node$/,
```

Comment:
this rule doesn't do anything since all the node_modules are ignored via the `nodeExternals` call above
(it does a whole lot for the bundling fork though)